src/utils/workspaceUtils.ts

```diff
@@ -22,9 +23,23 @@ export async function selectWorkspaceFile(context: IActionContext, placeHolder:
             };
         });
 
-        quickPicks.push({ label: `$(file-directory) ${localize('browse', 'Browse...')}`, description: '', data: undefined });
+        quickPicks.push(browseItem);
         input = await context.ui.showQuickPick(quickPicks, { placeHolder });
     }
 
     return input?.data || (await context.ui.showOpenDialog(options))[0].path;
 }
+
+export async function getRootWorkspaceFolder(placeHolder?: string): Promise<WorkspaceFolder | undefined> {
+    if (!workspace.workspaceFolders?.length) {
+        return undefined;
+    } else if (workspace.workspaceFolders?.length === 1) {
+        return workspace.workspaceFolders[0];
+    } else {
+        const folder = await window.showWorkspaceFolderPick({ placeHolder: placeHolder ?? localize('selectRootWorkspace', 'Select a folder for your workspace') });
```

Comment:
I'm not sure I get what this is asking me.

Select a folder where my project is basically?

Reply:
Maybe it should be worded differently, but this condition would be for multi-root workspaces.

Reply:
Ahhh, okay, that makes more sense.